.github/workflows/publish.yaml

```diff
@@ -13,15 +13,78 @@ on:
       - track/**
 
 jobs:
+  lib-check:
+    name: Check libraries
```

Comment:
Should we also run check libraries in the integration workflow? It seems more helpful there